src/index.js

```diff
@@ -36,6 +41,22 @@ class LeaderBoard extends React.Component {
     })
   }
 
+  recentPoints() {
+    let recentResults = fetch('https://fcctop100.herokuapp.com/api/fccusers/top/recent')
```

Comment:
This is can be fetched during the initial mount itself right? Is there any reason to fetch it every time.

Reply:
No I am not sure of how to do it. I will try.